neurovault/apps/statmaps/templates/statmaps/search.html

```diff
@@ -70,8 +70,8 @@
 
 		</script>
 		<!-- <gcse:search linkTarget="_self"></gcse:search> -->
-    <gcse:searchbox-only resultsUrl="http://192.168.99.103/search" queryParameterName="as_q"></gcse:searchbox-only>
-    <gcse:searchresults-only queryParameterName="as_q" linkTarget="_self"></gcse:searchresults-only>
+    <gcse:searchbox-only resultsUrl="http://neurovault.org/search"></gcse:searchbox-only>
```

Comment:
Is there a way to use a **relative** URL so it would work on both production and development?